verifyCredentials.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+module.exports = verify;
+
+/**
+ * Executes the verification logic by checking that fields are not empty using the provided apiKey.
+ *
+ * @param credentials object to retrieve apiKey from
+ *
+ * @returns Promise which resolves true
+ */
+
+const authTypes = {
+    BASIC: 'BASIC',
+}
+
+function verify(credentials) {
+    console.log('credentials:', JSON.stringify(credentials));
```

Comment:
do you want this to be here?

Reply:
it's copypaste from REST API component )